stock_move_location/wizard/stock_move_location_line.py

```diff
@@ -22,7 +22,9 @@ class StockMoveLocationWizardLine(models.TransientModel):
         string="Origin Location", comodel_name="stock.location"
     )
     destination_location_id = fields.Many2one(
-        string="Destination Location", comodel_name="stock.location"
+        string="Destination Location",
+        comodel_name="stock.location",
+        related="move_location_wizard_id.destination_location_id",
```

Comment:
Isn't this too restrictive as we maybe want to update some lines from the default value ?

Reply:
Finally I'm doing the field computed because it wasn't applying correctly the putaway strategies. The functionality is the same as before since this line field was readonly